src/people/utils/AssignedUnassignedBounties.tsx

```diff
@@ -156,9 +153,7 @@ const Bounties = (props: BountiesProps) => {
   return (
     <>
       {!!assignee?.owner_pubkey && !!assignee?.owner_alias ? (
-        <BountyLink
-          to={props.org_uuid ? `/bounty/${props.id}/${props.org_uuid}` : `/bounty/${props.id}`}
```

Comment:
looks like this `to` value was modified in this change

Reply:
I am going to resolve this one just give me few minutes